source/developers/content-modeling.rst

```diff
@@ -672,6 +672,38 @@ Model fields require their respective data type postfix as listed above.  The UI
 
 When setting up reserved variable names for your model, remember to remove the postfix auto-added by the UI since the variable name needs to be exactly the same as listed :ref:`above<reserved-variable-names>`.
 
+Please note that indexed ``text`` fields are case insensitive when performing a search, while ``string`` fields are case sensitive. If performing a case insensitive search on a ``string`` field is desired, Crafter CMS provides a way by enabling tokenization of the field in the content type.  To enable tokenization of a ``string`` field in Studio, put a check in the checkbox labeled **Tokenize for Indexing** in the properties section of the content type field.  Below is the ``Page - Article`` content type in a site created using the Website Editorial blueprint, showing the field ``Author`` with the ``Tokenize for Indexing`` option:
+
+.. image:: /_static/images/content-model/tokenize-for-indexing-property.jpg
+   :alt: Enable case insensitive keyword search for string fields in content type by clicking on "Tokenize for Indexing"
+   :width: 90%
+   :align: left
+
+|
+
+This will set the ``tokenize`` property of the field to ``true``, so for our example shown above,
+
+   .. code-block:: xml
```

Comment:
I don't think showing the form-definition snippet will be very helpful as most users are just familiar with the UI, how about replacing this with a small table to compare the results of a query against `_s` & `_t` fields? Something like:

Given the `author_s` field with a value of `Jane Doe`:

| Query    | Matches `author_s`? | Matches `author_t`? |
|----------|---------------------|---------------------|
| `Jane`     | No                  | Yes                 |
| `jane`     | No                  | Yes                 |
| `Jane Doe` | Yes                 | Yes                 |
| `jane doe` | No                  | Yes                 |